index.html

```diff
@@ -5007,13 +4989,281 @@ <h3 id="looping-AudioBufferSourceNode">
+            </li>
+          </ul>
+          <p>
+            The description of the algorithm is as follows:
+          </p>
+          <ol>
+            <li>Let <code>start</code> correspond to the <code>when</code>
+            parameter to <code>start()</code> if supplied, else 0.
+            </li>
+            <li>Let <code>stop</code> correspond to the sum of the
+            <code>when</code> and <code>duration</code> parameters to
+            <code>start()</code> if supplied, or the <code>when</code>
+            parameter to <code>stop()</code>, otherwise <em>Infinity</em>.
+            </li>
+            <li>Let <code>buffer</code> be the <a>AudioBuffer</a> employed by
+            the node.
+            </li>
+            <li>Let <code>bufferRate</code> be the sample rate employed by
+            <code>buffer</code>.
+            </li>
+            <li>Define a <em>playhead position</em> for <code>buffer</code> as
+            any quantity representing an exact, unquantized time offset in
+            seconds, relative to the time coordinate of the first sample frame
+            in the buffer. Playback rate and <a>AudioContext</a> sample rate
+            are not relevant: this offset is expressed purely in terms of the
+            buffer's audio content and <code>bufferRate</code>.
+            </li>
+            <li>Let <code>offset</code> correspond to the <code>offset</code>
+            parameter to <code>start()</code> if supplied, else 0. This is the
+            <em>playhead position</em> in the buffer at which playback will
+            begin.
+            </li>
+            <li>Let <code>loop</code>, <code>actualLoopStart</code> and <code>
+              actualLoopEnd</code> be the loop-related attributes of the node,
+              as discussed in the preceding section on looping. Note that the
+              loop start and loop end points are <em>playhead positions</em>,
+              not playback times.
+            </li>
+            <li>Let the function <code>playbackSignal(position)</code> be the
+            <em>playback signal function</em> for <code>buffer</code>, which is
+            a function that maps from a <em>playhead position</em> to a set of
+            output signal values, one for each output channel. If
+            <code>position</code> corresponds to the location of an exact
+            sample frame in the buffer, this function returns that frame.
+            Otherwise, its return value is determined by a UA-supplied
+            algorithm that interpolates between sample frames in the
+            neighborhood of <code>position</code>. If <code>position</code> is
+            greater than or equal to <code>actualLoopEnd</code> and there is no
+            subsequent sample frame in <code>buffer</code>, then the next
+            sample frame should be taken as the playback signal at
+            <code>actualLoopStart</code> for interpolation purposes.
```

Comment:
I don't think this captures what you want.  Plus I think it depends on the value of loop.  I think the only way position can be past loopEnd is if we're not looping, and then we do "normal" interpolation.

How the interpolation is done when we're looping and near the end (or beginning?) of the loop can be up to the UA?  Maybe playbackSignal needs an extra parameter to specify if we're looping or not.